### PR TITLE
Feature/integrated validation

### PR DESCRIPTION
**Pull request by tinoweiss**
Adds integrated validation to the application for validation within a browser. The enterID modal is replaced with an on-screen text box to be used to validate a device. The scan button has also been removed from the app when the app is run in a browser. No functionality changes when the app is run on a device.

### CODE REVIEW

File: client/src/app/views/details/details.controller.js [hunk 1]
```diff
@@ -180,7 +178,7 @@
                 message: 'Failed to checkin asset.'
             };
             $rootScope.errorModalText(err);
-            ModalService.get('enterIdModalDetails').open();
+            vm.switchView('validationView');
```
Comment (TimCreasman) [inline client/src/app/views/details/details.controller.js]: Why are there two cases for switching to the validation view? There were two open modal functions cause I needed to reopen the modal if the ID was invalid but does this serve a purpose when switching views?

Comment (tinoweiss) [inline client/src/app/views/details/details.controller.js]: You might be right that it's not needed on line 181. I'll double check that the flow works as expected for check in and check out failure and success and remove the switch to validationView here if it's not needed.

**Comment by TimCreasman** [comment]
Tweak the styling around the CheckIT logo on the home view so it is centered.

File: client/src/app/views/details/details.controller.js (hunk 1 above)
Comment (tinoweiss) [inline client/src/app/views/details/details.controller.js]: Looks like the modal or the validationView is not needed here. Will remove on next commit.

**Pull request merged**